src/encord_active/app/actions_page/export_filter.py

```diff
@@ -91,13 +91,24 @@ def filter_dataframe(df: pd.DataFrame) -> pd.DataFrame:
             key = f"filter-select-{column.replace(' ', '_').lower()}"
 
             if column == "tags":
+                # Enable filtering of frames without tags (containing '[]' values in the 'tags' column)
+                no_data_tag_label = "No data"
+                options = all_tags(get_state().project_paths) + [Tag(no_data_tag_label, TagScope.DATA)]
                 tag_filters = right.multiselect(
                     "Choose tags to filter",
-                    options=all_tags(get_state().project_paths),
+                    options=options,
                     format_func=lambda x: x.name,
                     key=key,
                 )
                 for tag in tag_filters:
+                    # Include frames without annotations if the 'no_tag' meta-tag was selected
+                    if tag.name == no_data_tag_label:
+                        data_rows = filtered[~filtered.is_label_metric]
+                        filtered_rows = [len(x) == 0 for x in data_rows["tags"]]
```

Comment:
Ah, because of the scenario where a frame has `data` tags but some of its labels don't have `label` tags. With that filter we prevent that a label without tags causes that its frame appears to have no tags.